src/cmds/scripts/pbs.service.in

```diff
@@ -55,7 +55,7 @@ DefaultDependencies=true
 [Service]
 Type=forking
 Restart=no
-TimeoutSec=5min
+TimeoutSec=0
```

Comment:
We don't know of a problem with an infinite timeout, which is why I changed it to that. All daemons handle cases where coming up is blocked by network or external environment; they "continue in background".